src/melee/characters.js

```diff
@@ -116,3 +142,17 @@ export function getCharacterShortName(externalCharacterId) {
   const character = getCharacterInfo(externalCharacterId) || {};
   return character.shortName;
 }
+
+// Return a human-readable color from a characterCode.
+export function getCharacterColorName(externalCharacterId, characterColor) {
+  const character = getCharacterInfo(externalCharacterId) || {};
+  if (!character || characterColor < 0 || characterColor > len(character)) {
```

Comment:
what does the len function do and what is its purpose here? Should you be comparing it to `character.colors` instead?

Reply:
A way to avoid doing this check might be to just use lodash's `get` function which will return undefined if the index is invalid

Reply:
Actually I'd probably just add `"Default"` to all of the arrays as index zero and then write the function as:
```
const character = getCharacterInfo(externalCharacterId) || {};
const colors = character.colors || [];
return colors[characterColor] || null;
```

Reply:
> what does the len function do and what is its purpose here? Should you be comparing it to `character.colors` instead?

looks like I forgot what language I was writing in for a minute